javascript/javascript1/week1/Housey-pricey.js

```diff
@@ -0,0 +1,33 @@
+console.log("Housey-pricey exercise");
+//peter's house//
+const houseWidth = 8;
+const houseDepth = 10;
+const houseHeight = 10;
+const Gardensize = 100;
+const ActualPrice = 2500000;
```

Comment:
It's good to keep a consistent variable name style. [For JavaScript is generally used camelCase](https://www.freecodecamp.org/news/snake-case-vs-camel-case-vs-pascal-case-vs-kebab-case-whats-the-difference/#heading-what-is-camel-case).
```suggestion
const gardenSize = 100;
const actualPrice = 2500000;
```